lib/bolt/node.rb

```diff
@@ -49,6 +49,9 @@ def initialize(host, port = nil, user = nil, password = nil, uri: nil,
       @sudo_password = transport_conf[:sudo_password]
       @run_as = transport_conf[:run_as]
       @tmpdir = transport_conf[:tmpdir]
+      @service_url = transport_conf[:service_url]
```

Comment:
I assume this will resolve to nil if this isn't in the config?

Reply:
it should